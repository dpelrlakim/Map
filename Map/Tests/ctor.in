newDefault a
newInit a
= b a
clear b
= a b
= c b
newInit c
newInit b
= b c
clear b
erase c 3
= b c
erase b 1
erase b 2
erase b 7
clear b
clear c
