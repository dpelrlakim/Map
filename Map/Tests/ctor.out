a:
a: (1,2) (3,4) (5,6) (7,8)
a: (1,2) (3,4) (5,6) (7,8)
b: (1,2) (3,4) (5,6) (7,8)
a: (1,2) (3,4) (5,6) (7,8)
b:
a:
b:
a:
b:
c:
a:
b:
c: (1,2) (3,4) (5,6) (7,8)
a:
b: (1,2) (3,4) (5,6) (7,8)
c: (1,2) (3,4) (5,6) (7,8)
a:
b: (1,2) (3,4) (5,6) (7,8)
c: (1,2) (3,4) (5,6) (7,8)
a:
b:
c: (1,2) (3,4) (5,6) (7,8)
a:
b:
c: (1,2) (5,6) (7,8)
a:
b: (1,2) (5,6) (7,8)
c: (1,2) (5,6) (7,8)
a:
b: (5,6) (7,8)
c: (1,2) (5,6) (7,8)
a:
b: (5,6) (7,8)
c: (1,2) (5,6) (7,8)
a:
b: (5,6)
c: (1,2) (5,6) (7,8)
a:
b:
c: (1,2) (5,6) (7,8)
a:
b:
c:
